content/_releases/23/31.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Product Update - v23.31
+description: Details of the product update v23.31 from 1st of August 2023.
+layout: rel
+version: "23.31"
+section: Product Updates
+category: product-updates
+releaseDate: 2023-08-01
+redirect_from:
+  - /releases/23.31.html
+---
+
+## New Features
+### Dashboard Redesign:
+* Added Ordinary Flows counter - *click on the counter to open `Flows` page with filtered ordinary flows*
+* Added Real-time Flows counter - *click on the counter to open `Flows` page with filtered real-time flows*
+* Redesigned the current Execution Statistics graph - *each day on the graph is clickable and leads users to the corresponding page*
+* Added new graph for Container errors - *each day on the graph is clickable and leads users to the corresponding page*
+* Transferred Workspace Memory Usage graph
```

Comment:
Transferred the QUOTA usage graph from the QUOTA page. QUOTA page is now deleted.

Reply:
Quota page is not deleted. We've transfered just one part - workspace usage with delatalized info per flow.

Reply:
Also we can add that when user clicks the workspace name in Workspace Memory Usage table at Current Month Quota
 page he/she will be redirected to the  Dashboard of corrwsponding workspace

Reply:
* Transferred the Workspace Quota Usage graph from the Quota page - *clicking the workspace name in the `Workspace Memory Usage` list on the `Current Month Quota` page will redirect the user to the corresponding workspace Dashboard.*